target_s3_jsonl/__init__.py

```diff
@@ -13,12 +13,13 @@
 
 from jsonschema import Draft4Validator, FormatChecker
 from decimal import Decimal
+from adjust_precision_for_schema import adjust_decimal_precision_for_schema
```

Comment:
That a great one. Thanks for adding it.

Reply:
Reading the [Handle multipleOf overflow](https://github.com/python-jsonschema/jsonschema/pull/746) fix introduced in [jsonschema v4.0.0](https://github.com/python-jsonschema/jsonschema/releases/tag/v4.0.0), I wonder if that update doesn't make `adjust_precision_for_schema` reduntant?